src/views/hidden-field.php

```diff
@@ -3,9 +3,26 @@
 		<?php
 		$hidden     = (array) get_post_meta( get_the_ID(), Tribe__Extension__Hide_Additional_Fields::$field_key, true );
 		$field_name = esc_attr( $customField['name'] );
+
+		/**
+		 * Allow filtering whether the checkboxes should be checked by default.
+		 *
+		 * @var bool $start_hidden
+		 */
+		$start_hidden = apply_filters( 'tribe_ext_hide_additional_fields_default_hidden', false );
+
+		if (
+			$start_hidden
+			&& get_post_status() == 'auto-draft'
```

Comment:
Let's use Yoda conditional
```suggestion
			&& 'auto-draft' === get_post_status()
```